### DISCUSSION TITLE
Version 0.4

### DISCUSSION THREAD

**Discussion by Mayitzin** [Mayitzin/ahrs#71] [General]
The version 0.4 is being completed and some of its novelties include:

- Unit Tests for the estimators and AHRS tools, including the attitude representations, metrics and geodesy models. See the subfolder `tests` in the latest commits and check them out.
- The documentation is re-arranged to simplify the access to the API and try to be clear about it. Besides, some functions lacking explanation are being properly described now.
- A better organized CI will take care of the proper tests to improve the quality of the code.
- Regarding quality, the code has been standarized to be coherent among estimators. And its quality is being checked with [codacy](https://app.codacy.com/gh/Mayitzin/ahrs/dashboard) with every new push or pull request in the repository.
- Submodules `plot` and `io` are removed from the code base.

I hope the newest version can satisfy the needs of every user in the easiest way. Future versions will include more estimators (and tools). This new version is targeted to improve its structure and growth.

Any comments are welcome.

**Comment by anisghaoui** [comment]
oh great! 
any news about the `to_angles` method?

**Comment by orestiskapar** [comment]
Hey there @Mayitzin. Thanks for your work on this. Any chance there might be a release to PyPI soon? 

There's been significant and very useful [changes since the last release](https://github.com/Mayitzin/ahrs/compare/v0.3.1..master), and it would be easier to manage this dependency with a package manager than through git modules.
